src/main/java/frc/robot/subsystems/FourBarArm.java

```diff
@@ -52,13 +56,15 @@ public void setArmSetpoint(double setpoint) {
 
   @Override
   public void periodic() {
-    // calculate the PID power level
-    double pidPower = m_pid.calculate(m_encoder.getPosition());
-    // calculate the feedforward power (nothing for now)
-    double feedforwardPower = 0.0;
+    if(m_enabled) {
+      // calculate the PID power level
+      double pidPower = m_pid.calculate(m_encoder.getPosition());
+      // calculate the feedforward power (nothing for now)
+      double feedforwardPower = 0.0;
```

Comment:
Hopefully write feedforward soon, but since this is beyond the scope of this PR, it's fine like this for now.